src/vs/workbench/contrib/webview/browser/pre/service-worker.js

```diff
@@ -100,7 +100,9 @@ class RequestStore {
 
 /**
  * Map of requested paths to responses.
- * @typedef {{ type: 'response', body: any, mime: string, etag: string | undefined, } | { type: 'not-modified', mime: string } | undefined} ResourceResponse
+ * @typedef {{ type: 'response', body: Uint8Array, mime: string, etag: string | undefined, mtime: number | undefined } |
```

Comment:
Note the stronger type on `body: Uint8Array`. AFAICT that's the only type which is ever passed through in a response and rely on it being so to call `Uint8Array.byteLength`.